server/src/main/java/org/jboss/as/server/mgmt/UndertowHttpManagementService.java

```diff
@@ -258,15 +258,18 @@ public synchronized void start(StartContext context) throws StartException {
                     }
                 }
             }
-        } catch (BindException e) {
-            final StringBuilder sb = new StringBuilder().append(e.getMessage());
-            if (bindAddress != null)
-                sb.append(" ").append(bindAddress);
-            if (secureBindAddress != null)
-                sb.append(" ").append(secureBindAddress);
-            throw new StartException(sb.toString(), e);
         } catch (Exception e) {
-            throw ServerLogger.ROOT_LOGGER.failedToStartHttpManagementService(e);
+            Throwable cause = e.getCause();
+            if (cause instanceof BindException) {
```

Comment:
More robust is 

if (e instanceof BindException || cause instanceof BindException) {